nasa-gibs/worldview
# Wv 2620 reactv18 upgrade

This updates the project from using React 17.0.2 to 18.0.0. The main change in Reactv18 is using `createRoot` for renders, which allows for concurrent rendering.

## How To Test
1. `git checkout wv-2620-reactv18-upgrade`
2. `npm ci`
3. `npm run watch`
4. Verify that behavior is consistent with previous versions.
